Erstes_Konzept/Erstes_Konzept.tex

```diff
@@ -51,5 +61,76 @@ \section{Behälter mit RFID Ausrüstung}
 \chapter{Zeitplan}
 
 \chapter{Finanzierungsplan}
+\section{Kosten}
+\subsection{Lese-/Suchbehälter}
+Die Daten von Feig wurden von Euro auf CHF umgerechnet (Wechselkurs 1.14). Zudem sind es die Kosten, welche man Zahlt würde ein 100er Stapel gekauft werden.
+
+\begin{tabularx}{\textwidth}{|r|X|r|r|}
+	\hline 
+	\textbf{Menge} & \textbf{Produkt} & \textbf{Kosten(CHF)} & \textbf{Kosten gesamt(CHF)} \\
+	\hline 
+	1 & RFID Reader ID ISC.LR2500-A (Feig) & 1'200 & 1'200 \\ 
+	\hline 
+	2 & RFID Multiplexer 8-fach HF Multiplexer (Feig) & 500 & 1'000 \\ 
+	\hline 
+	14 & RFID Antennen ID ISC.ANT800/600 (Feig)& 760 & 10'640 \\
+	\hline
+	16 & RFID Antennenkabel ID ISC.ANT.C-A (Feig) & 20 & 320 \\
+	\hline
+	1 & Netzteil ID NET.24V-B (Feig) & 30 & 30 \\
+	\hline
+	1 & Netzkabel ID CAB.NET.24V-B-EU (Feig) & 4 & 4 \\
+	\hline
+	1 & Rapberry PI 3 Model B+ & 40 & 40 \\
+	\hline
+	1 & SB-Kabel für Minicomputer Aukey CB-D11 (Aukey) & 15 & 15 \\
+	\hline
+	1 & Akku Goal Zero Yeti 400 Lithium (Goal Zero) & 1'000 & 1'000 \\
+	\hline
+	1 & Behälter 986417 (Kaiserkraft) & 39 & 39 \\
+	\hline
+	& & & \textbf{14'288} \\
+	\hline
+\end{tabularx} 
+\subsection{Umbau Roboter SW}
+Basierend auf den Stundenkosten der Firma bbv Software Services AG wurde die folgende Approximation errechnet.
+Die Stunden für die Umsetzung basieren auf Annahmen.
+
+\begin{tabularx}{\textwidth}{|r|r|X|r|}
+	\hline 
+	\textbf{Stunden(h)} & \textbf{Lohn(CHF/h)} & \textbf{Beschrieb} & \textbf{Kosten gesamt(CHF)} \\
+	\hline 
+	50 & 180 & Umsetzung, sofern die Software die Befehle bereits über das Netztwerk entgegennimmt. & 9'000 \\
+	\hline
+	200 & 180 & Umsetzung, sofern die Software komplexere Anpassung benötigt & 36'000 \\
+	\hline
+	8 & 200 & Allgemeine RE und weitere Kosten & 1'600 \\
+	\hline	
+	& & & \textbf{10'600 bis 37'600} \\ 
+	\hline
+\end{tabularx}
+\\
+Die Gesamtkosten können also zwischen 10'600.- und 37'600.- schwanken. Wobei diese Werte auf zwei Approximationen beruhen, und somit für deren Einhalt nicht garantiert werden kann.
+
+\subsection{Gesamt Rechnung}
+\begin{tabularx}{\textwidth}{|X|r|}
+	\hline
+	\textbf{Beschreibung} & \textbf{Kosten (CHF)} \\
+	\hline
+	RFID Lese-/Suchbehälter & 14'288 \\
+	\hline
+	Umbau Roboter SW & 10'600 bis 37'600 \\
+	\hline
+	Weitere BDA für Implementation & 1'000 \\
+	\hline
+	& \textbf{25'888 bis 52'888} \\
+	\hline
+\end{tabularx}
+
+\section{Kosten Manuelle Suche}
+Würde ein Exemplar deplatziert werden, würde eine Aufwändige suche beginnen, welche bei ca. 3 Millionen Exemplaren seine Zeit benötigen. Würde es Theoretisch möglich sein 1 Exemplar pro Sekunde zu durchsuchen bräuchte Man ca. 3 Millionen Sekunden. Dies Entspricht 50000 Minuten oder 833.32 Stunden oder 104.17 Arbeitstage à 8h oder knapp 5 Monaten. Würde nun ein Student mit einem Stundenlohn von ca. 20 CHF könnte diese Suche bis zu 16'666.67 CHF kosten. Würde die Suche von einem Arbeiter der Speicherbibliothek übernommen werden, könnten die Kosten noch höher ausfallen.
```

Comment:
Würde es **t**heoretisch möglich sein **ein** Exemplar  ...
... durchsuchen **,** bräuchte man **etwa drei** Millionen Sekunden.
Dies **e**ntspricht 50000 Minuten, 833.32 Stunden, 104.17 Arbeitstage à 8h oder knapp 5 Monate.
**Würde diese Arbeit von einer Student*in** mit einem Stundenlohn **von etwa** 20 CHF **durchgeführt**, könnte ...

Reply:
Generell, wollen wir diesen Abschnitt reinnehmen? Wegen Politik und so...